frappe/public/js/frappe/ui/tree.js

```diff
@@ -88,7 +89,7 @@ frappe.ui.Tree = class {
 			expandable: true,
 			is_root: true,
 			data: {
-				value: this.label
+				value: this.root_value
```

Comment:
If `root_value` is not passed when frappe.ui.Tree is initialised, we'll need to set it to `this.label` to prevent this from breaking.

Reply:
OK, I put in a check in the constructor whether root_value was null, and if so, set this.root_value to label. Hopefully that was the sort of thing you had in mind. Thanks for the feedback.